CombinatorialGames/Mathlib/CompRel.lean

```diff
@@ -0,0 +1,143 @@
+/-
+Copyright (c) 2025 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import Mathlib.Order.Antisymmetrization
+
+/-!
+# Comparability relation
+
+Two values in a preorder are comparable whenever `a ≤ b` or `b ≤ a`.
+
+## Main declarations
+
+* `CompRel`: The comparability relation. `CompRel r a b` means that `a` and `b` is related in
+  either direction by `r`.
+-/
+
+open Function
+
+variable {α : Type*} {a b c d : α}
+
+section Relation
+
+variable {r : α → α → Prop}
+
+/-- The comparability relation `CompRel r a b` means that either `r a b` or `r b a`. -/
+def CompRel (r : α → α → Prop) (a b : α) : Prop :=
+  r a b ∨ r b a
+
+theorem CompRel.of_rel {a b : α} (h : r a b) : CompRel r a b :=
```

Comment:
I should have noticed this earlier, but these variables were redundant.

Reply:
I fixed this in the Mathlib PR.